features/official_copy_legal_text.feature

```diff
@@ -0,0 +1,41 @@
+@official_copy
+Feature: Legal requirement updates made to Official Copy of the Register
```

Comment:
we agreed to do it another way for the moment.  I might reconsider this and go back to it.